src/server.js

```diff
@@ -248,8 +243,7 @@ function init(socket, client) {
 								const obj = {};
 
 								if (success) {
-									obj.success = "Successfully updated your password, all your other sessions were logged out";
-									obj.token = client.config.token;
+									obj.success = "Successfully updated your password";
```

Comment:
You added that behavior in https://github.com/thelounge/lounge/pull/370, and that was a good thing for security purposes.

#986 currently logs out active connections, it will need to be updated to log out inactive connections too (since this PR helps keeping track of connections even after you close The Lounge).

Until #986 happens, we really really should not remove this, otherwise there is no way to log out from other sessions anymore, which was possible before.